requirements.txt

```diff
@@ -7,3 +7,4 @@ raven>=5.23.0
 psutil>=3.0.0
 zipstream>=1.1.4
 typing>=3.5.3.0 # Otherwise yarl fail with python 3.4
+prompt-toolkit
```

Comment:
I don't think so, actually cannot see any backlog of versions (to choose the minimal).